### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by someyaaltous**

### CODE REVIEW

**Review by SajaLahaleeh — APPROVED** [review]

**Review by DuaaH — APPROVED** [review]

**Review by Muniralsharif — APPROVED** [review]
me and somya do this

**Comment by DuaaH** [comment]
me and saja see this changes

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+  - router.js
+  - handler.js
+  - database folder:
+    - db_build.js
+    - db_build.sql
+    - db_connection.js
+  - queries folder:
+    - getData.js
+    - postUser.js
+- config.envPublic folder
```
Comment (jema28) [inline README.md]: There's an error here, please delete public folder on this last line

**Review by jema28 — CHANGES_REQUESTED** [review]

**Review by DuaaH — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (someyaaltous) [reply README.md]: @jema28 we fix it

**Review by jema28 — APPROVED** [review]

**Pull request merged**